addons/website_profile/views/website_profile.xml

```diff
@@ -59,7 +59,7 @@
                                 <div class="dropdown-menu dropdown-menu-right w-100" style="right: 10px;">
                                     <form class="px-3" t-attf-action="/profile/users" role="search" method="get">
                                         <div class="input-group">
-                                            <input type="text" class="form-control" name="search" placeholder="Search courses"/>
+                                            <input type="text" class="form-control" name="search" placeholder="Search users"/>
```

Comment:
No translation to update in 'website_profile' module ?

Reply:
hi @tde-banana-odoo ,
for the website_profile, the term is already existing in the pot file